### PR TITLE
[Merged by Bors] - feat: port Algebra.Category.FGModuleCat.Limits

### PR DESCRIPTION
**Pull request by int-y1**
---

todo:

* [ ] first instance has an error

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/!4.235519.20-.20Algebra.2ECategory.2EFGModuleCat.2ELimits

File: Mathlib/Algebra/Category/ModuleCat/Products.lean [hunk 1]
```diff
@@ -26,7 +26,7 @@ namespace ModuleCat
 
 variable {R : Type u} [Ring R]
 
-variable {ι : Type v} (Z : ι → ModuleCat.{v} R)
+variable {ι : Type v} (Z : ι → ModuleCatMax.{v, w} R)
```
Comment (semorrison) [inline Mathlib/Algebra/Category/ModuleCat/Products.lean:29]: This was just misported in #3515, it seems.

**Review by jcommelin — APPROVED** [review]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**